editor/plugins/path_2d_editor_plugin.cpp

```diff
@@ -531,42 +530,42 @@ Path2DEditor::Path2DEditor(EditorNode *p_editor) {
 	CanvasItemEditor::get_singleton()->add_control_to_menu_panel(base_hb);
 
 	sep = memnew(VSeparator);
+
```

Comment:
That separation line should be before `curve_edit = memnew(Button);` no? Instead of separating the instantiation of `sep` and its `add_child` call.